From cd9d8a7a6a8cf4e7d949c878e30a4afa4c5b1e4e Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Tue, 11 Feb 2025 10:52:45 +0800
Subject: [PATCH 12/14] Skip writing symbol cache to non-writable directories

---
 src/moduleutils/PhutilLibraryMapBuilder.php | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/moduleutils/PhutilLibraryMapBuilder.php b/src/moduleutils/PhutilLibraryMapBuilder.php
index 38a1bd21d49a..dac03646074f 100644
--- a/src/moduleutils/PhutilLibraryMapBuilder.php
+++ b/src/moduleutils/PhutilLibraryMapBuilder.php
@@ -196,6 +196,11 @@ final class PhutilLibraryMapBuilder extends Phobject {
    */
   private function writeSymbolCache(array $symbol_map, array $source_map) {
     $cache_file = $this->getPathForSymbolCache();
+    try {
+      Filesystem::assertWritableFile($cache_file);
+    } catch (Exception $ex) {
+      return;
+    }
 
     $cache = array(
       self::SYMBOL_CACHE_VERSION_KEY => self::SYMBOL_CACHE_VERSION,
-- 
2.48.1

