From 0bea9b4f7d3d07908a9c85472ffe9563fe091d4f Mon Sep 17 00:00:00 2001
From: Jiajie Chen <c@jia.je>
Date: Sun, 4 Feb 2024 22:19:17 +0800
Subject: [PATCH 2/5] AOSCOS: helgrind race supp

Signed-off-by: Jiajie Chen <c@jia.je>
---
 glibc-2.X-helgrind.supp.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/glibc-2.X-helgrind.supp.in b/glibc-2.X-helgrind.supp.in
index 9b1ef9ae4..c447b3a55 100644
--- a/glibc-2.X-helgrind.supp.in
+++ b/glibc-2.X-helgrind.supp.in
@@ -118,6 +118,12 @@
    fun:mythread_wrapper
    obj:@GLIBC_LIBPTHREAD_PATH@
 }
+{
+   helgrind-glibc2X-102a
+   Helgrind:Race
+   fun:mythread_wrapper
+   obj:*vgpreload_helgrind*.so
+}
 {
    helgrind-glibc2X-103
    Helgrind:Race
-- 
2.47.1.windows.1

