From ef8c237dd5633cb15eec7ab478432d9fa6c11489 Mon Sep 17 00:00:00 2001
From: Jiangjin Wang <kaymw@aosc.io>
Date: Sun, 21 Jan 2024 01:01:53 -0800
Subject: [PATCH] explicitly set cpp_eh to default

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index f0e56521c..bea255344 100644
--- a/meson.build
+++ b/meson.build
@@ -1,7 +1,7 @@
 project('Aegisub', ['c', 'cpp'],
         license: 'BSD-3-Clause',
         meson_version: '>=0.57.0',
-        default_options: ['cpp_std=c++20', 'buildtype=debugoptimized', 'harfbuzz:icu=disabled', 'ffmpeg:libdav1d=enabled'],
+        default_options: ['cpp_std=c++20', 'buildtype=debugoptimized', 'harfbuzz:icu=disabled', 'ffmpeg:libdav1d=enabled', 'cpp_eh=default'],
         version: '3.4.2')
 
 cmake = import('cmake')
-- 
2.48.1

