From ecc2df2ada1269f842d65c8cfc3750a585ff4318 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 27 Dec 2024 14:08:50 +0800
Subject: [PATCH 3/4] ARCHLINUX: install sensord

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 47ffe788..0142abb2 100644
--- a/Makefile
+++ b/Makefile
@@ -80,7 +80,7 @@ MANDIR := $(PREFIX)/man
 ARCH := $(firstword $(subst -, ,$(shell $(CC) -dumpmachine)))
 
 # Extra non-default programs to build; e.g., sensord
-#PROG_EXTRA := sensord
+PROG_EXTRA := sensord
 
 # Build and install static library
 BUILD_STATIC_LIB := 1
-- 
2.47.1

