From 198e9498f0c89341720f7a6ed18e0f3890ea938f Mon Sep 17 00:00:00 2001
From: Kaiyang Wu <self@origincode.me>
Date: Wed, 29 Jan 2025 18:50:09 -0800
Subject: [PATCH 3/3] metadata-editor: fix incompatible pointer type

Signed-off-by: Kaiyang Wu <self@origincode.me>
---
 plug-ins/metadata/metadata-editor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plug-ins/metadata/metadata-editor.c b/plug-ins/metadata/metadata-editor.c
index 87a2e87098..bc02b44020 100644
--- a/plug-ins/metadata/metadata-editor.c
+++ b/plug-ins/metadata/metadata-editor.c
@@ -2140,7 +2140,7 @@ metadata_dialog_editor_set_metadata (GExiv2Metadata *metadata,
                             }
                           else
                             {
-                              if (! g_strv_contains (values, equiv_values[evi]))
+                              if (! g_strv_contains ((const gchar* const*) values, equiv_values[evi]))
                                 {
                                   gchar *tmpvalue;
 
-- 
2.48.1

