From cc627d20fec800e69796088e17d91f00b0ea47d6 Mon Sep 17 00:00:00 2001
From: eatradish <sakiiily@aosc.io>
Date: Thu, 28 Nov 2024 11:09:59 +0800
Subject: [PATCH] cargo: use default --release build settings

---
 Cargo.toml | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 98922a7ca2..6bc3155309 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -604,20 +604,6 @@ ui_input = { codegen-units = 1 }
 vcs_menu = { codegen-units = 1 }
 zed_actions = { codegen-units = 1 }
 
-[profile.release]
-debug = "limited"
-lto = "thin"
-codegen-units = 1
-
-[profile.release.package]
-zed = { codegen-units = 16 }
-
-[profile.release-fast]
-inherits = "release"
-debug = "full"
-lto = false
-codegen-units = 16
-
 [workspace.lints.clippy]
 dbg_macro = "deny"
 todo = "deny"
-- 
2.47.1

