From ffa0a04a60870a361f82f3661bf0485d5e853563 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 27 Dec 2024 18:11:06 +0800
Subject: [PATCH] AOSCOS: configure.ac: support detecting neon 0.33.x

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 4f86cde..ede9b1e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -36,7 +36,7 @@ AC_PROG_LN_S
 # Checks for libraries.
 AM_GNU_GETTEXT_VERSION(0.19.8)
 AM_GNU_GETTEXT([external])
-NE_REQUIRE_VERSIONS([0], [27 28 29 30 31 32])
+NE_REQUIRE_VERSIONS([0], [27 28 29 30 31 32 33])
 DAV_CHECK_NEON
 
 # Checks for header files.
-- 
2.47.1

