From 4a202ecebbe140985d20378d933f1eda1557845a Mon Sep 17 00:00:00 2001
From: eatradish <sakiiily@aosc.io>
Date: Fri, 5 Jul 2024 21:21:23 +0800
Subject: [PATCH 2/2] hack: fix IME input text

---
 bin_steam.sh | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/bin_steam.sh b/bin_steam.sh
index 49f9d8a..04e6be0 100755
--- a/bin_steam.sh
+++ b/bin_steam.sh
@@ -303,4 +303,10 @@ fi
 # go to the install directory and run the client
 cd "$LAUNCHSTEAMDIR"
 
+# Setting IME to fcitx5 to fix input
+export GTK_IM_MODULE=fcitx5
+export QT_IM_MODULE=fcitx5
+export QT5_IM_MODULE=fcitx5
+export XMODIFIERS=@im=fcitx5
+
 exec "$LAUNCHSTEAMDIR/$STEAMBOOTSTRAP" ${log_opened+-srt-logger-opened} "$@"
-- 
2.47.1

