From c1814fa841b5095f549b34d1ddfc8a56dd841195 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Thu, 30 Jan 2025 09:34:18 +0800
Subject: [PATCH 1/2] AOSCOS: Pin data directory

---
 etc/astrometry.cfg-dist | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/astrometry.cfg-dist b/etc/astrometry.cfg-dist
index 4e5c3c22f6b3..2777a450819b 100644
--- a/etc/astrometry.cfg-dist
+++ b/etc/astrometry.cfg-dist
@@ -26,7 +26,7 @@
 cpulimit 300
 
 # In which directories should we search for indices?
-add_path DATA_INSTALL_DIR
+add_path /usr/share/astrometry/data
 
 # Load any indices found in the directories listed above.
 autoindex

base-commit: 3f5bd4fa3226a63faf06b1f8ec5ac4c7b59b1576
-- 
2.48.1

