diff --git a/src/bin/keactrl/kea-ctrl-agent.conf.pre b/src/bin/keactrl/kea-ctrl-agent.conf.pre
index d8e0429..b13b723 100644
--- a/src/bin/keactrl/kea-ctrl-agent.conf.pre
+++ b/src/bin/keactrl/kea-ctrl-agent.conf.pre
@@ -32,15 +32,15 @@
     "control-sockets": {
         "dhcp4": {
             "socket-type": "unix",
-            "socket-name": "/tmp/kea4-ctrl-socket"
+            "socket-name": "/run/kea/kea4-ctrl-socket"
         },
         "dhcp6": {
             "socket-type": "unix",
-            "socket-name": "/tmp/kea6-ctrl-socket"
+            "socket-name": "/run/kea/kea6-ctrl-socket"
         },
         "d2": {
             "socket-type": "unix",
-            "socket-name": "/tmp/kea-ddns-ctrl-socket"
+            "socket-name": "/run/kea/kea-ddns-ctrl-socket"
         }
     },
 
@@ -73,11 +73,12 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of the file
-                "output": "@localstatedir@/log/kea-ctrl-agent.log"
+                // "output": "@localstatedir@/log/kea-ctrl-agent.log"
+                "output": "syslog",
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
-                // "pattern": "%-5p %m\n"
+                "pattern": "%-5p %m\n"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
diff --git a/src/bin/keactrl/kea-dhcp-ddns.conf.pre b/src/bin/keactrl/kea-dhcp-ddns.conf.pre
index b75b51f..a7c5a43 100644
--- a/src/bin/keactrl/kea-dhcp-ddns.conf.pre
+++ b/src/bin/keactrl/kea-dhcp-ddns.conf.pre
@@ -23,7 +23,7 @@
   "port": 53001,
   "control-socket": {
       "socket-type": "unix",
-      "socket-name": "/tmp/kea-ddns-ctrl-socket"
+      "socket-name": "/run/kea/kea-ddns-ctrl-socket"
   },
   "tsig-keys": [],
   "forward-ddns" : {},
@@ -44,11 +44,12 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of the file
-                "output": "@localstatedir@/log/kea-ddns.log"
+                // "output": "@localstatedir@/log/kea-ddns.log"
+                "output": "syslog",
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
-                // "pattern": "%-5p %m\n"
+                "pattern": "%-5p %m\n"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
diff --git a/src/bin/keactrl/kea-dhcp4.conf.pre b/src/bin/keactrl/kea-dhcp4.conf.pre
index 55af9db..96bcbb8 100644
--- a/src/bin/keactrl/kea-dhcp4.conf.pre
+++ b/src/bin/keactrl/kea-dhcp4.conf.pre
@@ -49,7 +49,7 @@
     // more. For detailed description, see Sections 8.8, 16 and 15.
     "control-socket": {
         "socket-type": "unix",
-        "socket-name": "/tmp/kea4-ctrl-socket"
+        "socket-name": "/run/kea/kea4-ctrl-socket"
     },
 
     // Use Memfile lease database backend to store leases in a CSV file.
@@ -436,11 +436,12 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of the file
-                "output": "@localstatedir@/log/kea-dhcp4.log"
+                // "output": "@localstatedir@/log/kea-dhcp4.log"
+                "output": "syslog",
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
-                // "pattern": "%-5p %m\n",
+                "pattern": "%-5p %m\n"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
diff --git a/src/bin/keactrl/kea-dhcp6.conf.pre b/src/bin/keactrl/kea-dhcp6.conf.pre
index 271021b..3173ba4 100644
--- a/src/bin/keactrl/kea-dhcp6.conf.pre
+++ b/src/bin/keactrl/kea-dhcp6.conf.pre
@@ -43,7 +43,7 @@
     // description, see Sections 9.12, 16 and 15.
     "control-socket": {
         "socket-type": "unix",
-        "socket-name": "/tmp/kea6-ctrl-socket"
+        "socket-name": "/run/kea/kea6-ctrl-socket"
     },
 
     // Use Memfile lease database backend to store leases in a CSV file.
@@ -395,11 +395,12 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of the file
-                "output": "@localstatedir@/log/kea-dhcp6.log"
+                // "output": "@localstatedir@/log/kea-dhcp6.log"
+                "output": "syslog",
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
-                // "pattern": "%-5p %m\n",
+                "pattern": "%-5p %m\n"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
diff --git a/src/bin/keactrl/kea-netconf.conf.pre b/src/bin/keactrl/kea-netconf.conf.pre
index c8a3878..5bce47c 100644
--- a/src/bin/keactrl/kea-netconf.conf.pre
+++ b/src/bin/keactrl/kea-netconf.conf.pre
@@ -30,13 +30,13 @@
         "dhcp4": {
             "control-socket": {
                 "socket-type": "unix",
-                "socket-name": "/tmp/kea4-ctrl-socket"
+                "socket-name": "/run/kea/kea4-ctrl-socket"
             }
         },
         "dhcp6": {
             "control-socket": {
                 "socket-type": "unix",
-                "socket-name": "/tmp/kea6-ctrl-socket"
+                "socket-name": "/run/kea/kea6-ctrl-socket"
             }
         }
     },
@@ -69,11 +69,12 @@
                 // - syslog (logs to syslog)
                 // - syslog:name (logs to syslog using specified name)
                 // Any other value is considered a name of a time
-                "output": "@localstatedir@/log/kea-netconf.log"
+                // "output": "@localstatedir@/log/kea-netconf.log"
+                "output": "syslog",
 
                 // Shorter log pattern suitable for use with systemd,
                 // avoids redundant information
-                // "pattern": "%-5p %m\n"
+                "pattern": "%-5p %m\n"
 
                 // This governs whether the log output is flushed to disk after
                 // every write.
